File: Latihan4/Py/nilai.py
nilai = int(input("Masukan Nilai = "))

if nilai >= 80 and nilai <= 100:
    print("Nilai A")
elif nilai >= 65 and nilai <= 79:
    print("Nilai B")
elif nilai >= 50 and nilai <= 64:
    print("Nilai C")
elif nilai >= 35 and nilai <= 49:
    print("Nilai D")
elif nilai >= 0 and nilai <= 34:
    print("Nilai E")
else:
    print("404 Not Found")

File: Latihan4/C++/nilai.cpp
#include <iostream>

using namespace std;

int main()
{
    int nilai;

    cout << "Masukan Nilai = ";
    cin >> nilai;
    if (nilai >= 80 && nilai <= 100)
    {
        cout << "Nilai A";
    }
    else if (nilai >= 65 && nilai <= 79)
    {
        cout << "Nilai B";
    }
    else if (nilai >= 50 && nilai <= 64)
    {
        cout << "Nilai C";
    }
    else if (nilai >= 35 && nilai <= 49)
    {
        cout << "Nilai D";
    }
    else if (nilai >= 0 && nilai <= 34)
    {
        cout << "Nilai E";
    }
    else
    {
        cout << "404 Not Found";
    }
    

    return 0;
}
